lib/brewdler/utils.rb

```diff
@@ -1,10 +1,29 @@
 module Brewdler
   def self.system cmd, *args
-    pid = fork do
-      $stdout.reopen("/dev/null")
-      exec(cmd, *args)
+    verbose = ARGV.verbose?
+    rd, wr = IO.pipe
+    logs = []
+    begin
+      pid = fork do
+        rd.close
+        $stdout.reopen(wr)
+        wr.close
+        exec(cmd, *args)
+      end
+
+      wr.close
+      while buf = rd.gets
+        puts buf if verbose
+        logs << buf
+      end
+    rescue
+      rd.close
     end
```

Comment:
@mikemcquaid Have you working on this using `IO.popen`?

I planned to implement `brew brewdle cleanup` in this way:
Merge this PR -> new PR to decouple parsing Brewfile and actual installation -> `brew brewdle cleanup`